repos_info.json

```diff
@@ -120,12 +120,15 @@
       "name": "ocw-to-hugo",
       "repo_url": "https://github.com/mitodl/ocw-to-hugo.git",
       "channel_name": "ocw-to-hugo",
-      "project_type": "web_application",
-      "web_application_type": "hugo",
-      "ci_hash_url": "https://ocw-next.netlify.app/static/ocw-to-hugo-hash.txt",
-      "rc_hash_url": "https://ocwnext-rc.odl.mit.edu/static/ocw-to-hugo-hash.txt",
-      "prod_hash_url": "https://ocwnext.odl.mit.edu/static/ocw-to-hugo-hash.txt",
-      "announcements": false
+      "project_type": "library",
+      "packaging_tool": "npm",
+      "announcements": false,
+      "update_other_repos": [
```

Comment:
The functionality surrounding updating other repos looks good, and it's good we've generalized it but I'm not sure `ocw-to-hugo` is the target I had in mind for it.  `ocw-to-hugo` should be published to NPM but I don't think we want to automatically update `ocw-course-hugo-theme` to use the latest version.  Rather, as we did in `hugo-course-publisher`, I think it would be best to implement the version update in the PR you make in the theme to support the changes you are committing.  I may be misinterpreting something here though, does this just open a PR with the update?

Reply:
It makes a commit which would be part of the release PR for ocw-course-hugo-theme. It's a good point though that it may be more flexible to have the version update be part of the PR which updates other parts of ocw-course-hugo-theme. I'll leave out this configuration setting and if we change our minds in the future we can add it back in

Reply:
That's exactly what I was thinking, do a release in `ocw-to-hugo` and then `@doof publish`, then in your PR that adds support for the new functionality you update the version.  That way, if a new build of the site happens after the `ocw-to-hugo` release but **before** the `ocw-course-hugo-theme` PR that supports it, the build will still use the older version of `ocw-to-hugo` and continue to function properly.